### ISSUE TITLE
Fulltext search of OCR content: Implement in-document search

### ISSUE THREAD

**Issue by marlo-longley** [sul-dlss/vt-arclight#155]
Ref #146 

This will rely on content-search for an in-viewer experience. More analysis needed.

**Comment by thatbudakguy** [comment]
@marlo-longley what do you think about moving this out of the workcycle? is it a crucial piece of full-text search, or could we consider full-text search as MVP without it? I think it will be hard to deliver this before launch.

**Comment by laurensorensen** [comment]
Adding design needed so I can get @ggeisler 's take on this - is it a "must have" or "nice to have"? Thanks!!

**Comment by thatbudakguy** [comment]
blocked by https://github.com/projectblacklight/arclight/pull/1379
